cudaaligner/benchmarks/singlealignment/main.cpp

```diff
@@ -10,9 +10,9 @@
 
 #include <benchmark/benchmark.h>
 
-#include "utils/genomeutils.hpp"
-#include "cudautils/cudautils.hpp"
-#include "cudaaligner/aligner.hpp"
+#include "claragenomics/utils/genomeutils.hpp"
```

Comment:
The new includes look much nicer in my opinion.

But nit-picky me has another request: Could you use `#include <>` for all public headers, and `#include ""` only for internal headers? :)

Reply:
Actually, I just noticed, that we decided to make these files private. So you only need to change the file below.
Somehow having them in `""` feels funny. For most projects I worked on all files which required the compiler to get a `-I<directory>` were included via `<>` and those were usually the public headers. This is the first time that these two criteria do not coincide, because the project is somewhat divided into subprojects.

Reply:
Indeed as I was making these changes I was bothered by the mixture of `""` and `<>`, so I started making the changes locally and then got distracted by other things XD.
So far I haven't pushed the changes to make the `utils` folders private. So I'm making all of them use `<>` for now. This led me to think that even the private headers will probably get added by the `-I` directive in the compiler based on how `CMake` sets them up. I was thinking of reserving the `""` only for the headers included with relative paths (like local files). What do you think of this?

Reply:
Another point that struck me here is that otherwise whenever we make one module public to private, or vice versa, we'll have to go in and change `""` or `<>` in its include files...

Reply:
Good point, although that will prevent us from making too many irrational decisions. ;) Anyhow, yes, I like the idea of local/relative path vs. -I files. Makes sense.

Reply:
Done!